### DISCUSSION TITLE
Is Storybook Webpack 5 stable?

### DISCUSSION THREAD

**Discussion by phuctm97** [storybookjs/storybook#18602] [Help] [answered]
Hi, I'm wondering if Storybook Webpack 5 is still in experimental or already stable? All documentation seems to say it is still experimental.

**Comment by shilman** [comment, marked as answer]
Yes, it's fairly stable at this point. Any chance you can submit a PR to update the documentation? Or point us at the outdated docs?

The biggest source of issues is that we still ship webpack4 by default and there are sometimes hoisting issues/dependency conflicts. In the next release, 7.0, Storybook will no longer have a default builder and this problem will go away.

**Reply by phuctm97** [reply]
That is awesome to hear. Where can I follow progress of 7.0?

I will create the issue/PR later. In the meantime, it would be great just to have a page that list all known issues of Webpack 5 so that developers can make the decisions. It is fairly unknown and I just decided to stick with Webpack 4 at the moment.

**Reply by shilman** [reply]
One thing you know is that it's likely Webpack4 will go away in 7.0. We've already removed it in the early alphas.

You can follow the progress of 7.0 here: https://github.com/storybookjs/storybook/releases

As this starts to stabilize we will start publishing blog posts about the changes ahead: https://storybook.js.org/blog/

There are some specs/docs about different projects we're working on here, but it's all in flux https://www.notion.so/chromatic-ui/Storybook-7-0-WIP-6a9a64c5fe5f4cc08a8357eee9e7ce3f
